spicepy/_client.py

```diff
@@ -102,25 +115,51 @@ def __init__(
         api_key: str,
         flight_url: str = "grpc+tls://flight.spiceai.io",
         firecache_url: str = "grpc+tls://firecache.spiceai.io",
+        http_url: str = "https://data.spiceai.io",
         tls_root_cert: Union[str, Path, None] = None,
-    ):
+    ):  # pylint: disable=R0913
         tls_root_certs = _Cert(tls_root_cert).tls_root_certs
         self._flight = _SpiceFlight(flight_url, api_key, tls_root_certs)
         self._firecache = _SpiceFlight(firecache_url, api_key, tls_root_certs)
 
+        self.api_key = api_key
+        self.http = HttpRequests(http_url, self._headers())
+
+    def _headers(self) -> Dict[str, str]:
+        return {
+            "X-API-Key": self._api_key(),
+            "Accept": "application/json",
+            "User-Agent": "spicepy 0.1",
+        }
+
+    def _api_key(self) -> str:
+        key = self.api_key
+        if key is None:
+            key = os.environ.get("SPICEAI_API_KEY")
```

Comment:
Is this environment variable consistent? It looks a bit weird. I would expect `SPICE_API_KEY` or just `API_KEY`